README.md

```diff
@@ -24,7 +24,41 @@ pip install rasa_core_sdk
 ## Usage
 
 Detailed instructions can be found in the Rasa Core Documentation about 
-[Custom Actions](https://rasa.com/docs/core/customactions.html).
+[Custom Actions](https://rasa.com/docs/core/customactions).
+
+## Docker 
+
+### Usage
+
+In order to start an action server using implemented custom actions, you can use the available Docker image `rasa/rasa_core_sdk:latest`.
+
+Before starting the action server ensure that the folder containing your actions is handled as Python module and therefore has to contain a file called `__init__.py`
+
+Then start the action server using:
+
+```bash
+docker run --mount type=bind,source=<ABSOLUTE_PATH_TO_YOUR_ACTIONS>,target=/app/actions \
+	rasa/rasa_core_sdk:latest 
+```
+
+The action server is then avaible at `http://localhost:5055/webhook`. 
```

Comment:
aren't we missing a port mapping from inside the container to the host machine? otherwise this webhook can't be reached

Reply:
added port mapping